### ISSUE TITLE
add an -oA option to save in all formats like nmap.

### ISSUE THREAD

**Issue by gprime31** [robertdavidgraham/masscan#733]
add an -oA option to save in all formats like nmap.

**Comment by mzpqnxow** [comment]
> add an -oA option to save in all formats like nmap.

I agree this could be a nice convenience for some but the best way to accomplish itin my opinion is to always write to the binary format (-oB) which is lossless and efficient on disk

Then, afterwards, convert as needed to each format using:

    masscan --readscan out.bin -oD out.ndjson
    masscan --readscan out.bin -oX out.xml
    ...
